creator-node/src/routes/tracks.js

```diff
@@ -550,26 +551,101 @@ module.exports = function (app) {
       return sendResponse(req, res, errorResponseBadRequest(`Invalid ID: ${encodedId}`))
     }
 
-    const { blockchainId: blockchainIdFromTrack } = await models.Track.findOne({
-      attributes: ['blockchainId'],
-      where: { blockchainId },
-      order: [['clock', 'DESC']]
-    })
-
-    if (!blockchainIdFromTrack) {
-      return sendResponse(req, res, errorResponseBadRequest(`No track found for blockchainId ${blockchainId}`))
-    }
-
-    const { multihash } = await models.File.findOne({
+    let fileRecord = await models.File.findOne({
       attributes: ['multihash'],
       where: {
         type: 'copy320',
-        trackBlockchainId: blockchainIdFromTrack
+        trackBlockchainId: blockchainId
       },
       order: [['clock', 'DESC']]
     })
 
-    if (!multihash) {
+    if (!fileRecord) {
```

Comment:
nice, maybe safer to do a `if (!fileRecord || !fileRecord.multihash)` for consistency with check on line 648 below

Reply:
don't feel too strongly but i don't see how that helps here. if file record doesn't exist thats what the rest of the stuff is meant to do. if fileRecord does exist but multihash doesn't, we're boned. the next part is a hail mary anyway.

Reply:
yep agreed